frontend/index.html

```diff
@@ -16,13 +16,17 @@
     <link rel="shortcut icon" type="image/png" href="/images/ode_logo_180x180.png" sizes="180x180">
     <link rel="shortcut icon" type="image/png" href="/images/ode_logo_192x192.png" sizes="192x192">
 
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Exo+2&display=swap" rel="stylesheet">
```

Comment:
I think it's better to download the fonts you want to use and to include them in our `fonts` directory.
We don't want to depend on another website (which can be down or slow, even if it's Google), and people who don't want to be tracked by Google (fonts come with associated cookies) might filter Google CDN.

(Though, the "Roboto" folder might be cleaned, I don't think we need all those versions of the font)

Reply:
I understand, I'll will add it to the repo.

For "Roboto" I didn't find any use of this font files (in `@font-face`) so maybe all of it should be removed?